- Created models file in app/models

bishoyAtif opened the pull request:
- Created Sqlite database called kawn.db
- Added the meta class and associated it with the database
- Added a datefield to the base class
- Added "jquery.js" to /static to use it later in moment library to show the date in templates

bishoyAtif committed:
- Created models file in app/models
- Created Sqlite database called kawn.db
- Added the meta class and associated it with the database
- Added a datefield to the base class

mxmaher commented:
It's good that you thought of adding jquery but it's not it's time yet
the date field name should be created_on or something similar 
 and the name of the models file should change maybe it should  be core as it will contain the core models of the system

mxmaher commented:
You also need to use a 4 spaces indentation level
it's the standards after all

bishoyAtif committed:
- modified the base model name to core
- edited the indentation with spaces "sorry for this trivial mistake, I switched my editor to sublime text"
